src/js/components/Beacon/BeaconCommon/Filters.tsx

```diff
@@ -29,37 +29,45 @@ const NetworkFilterToggle = () => {
   );
 };
 
-// ideally:
-// - should not permit you to make multiple queries on the same key (Redmine #1688)
-
 const BUTTON_STYLE = { margin: '10px 0' };
 
 const Filters = ({ filters, setFilters, form, beaconFiltersBySection, isNetworkQuery }: FiltersProps) => {
   const t = useTranslationFn();
+  const [filterIndex, setFilterIndex] = useState<number>(1);
 
   const { maxQueryParameters: maxFilters, maxQueryParametersRequired } = useConfig();
```

Comment:
katsu is supposed to return max int for max query params if the user has `query:data`, so I don't think this `maxQueryParametersRequired` state is actually needed

Reply:
No trouble to remove this, but is it needed in all the other places it's used?

Reply:
It's only used by Beacon as far as I can tell